.github/workflows/build-package-test.yml

```
@@ -12,8 +12,8 @@ env:
   CURRENT_BRANCH: ${GITHUB_REF##*/}
   TEST: true
 on:
-  push:
-    branches: "**"
+#  push:
+#    branches: "**"
```

We should uncomment these lines before merging the PR.

```suggestion
  push:
    branches: "**"
```

---

Uncommented